Frameworks/UIKit/UIViewController.mm

```diff
@@ -2148,6 +2103,17 @@ - (UIModalPresentationStyle)modalPresentationStyle {
 }
 
 /**
+ @Status Caveat
+ @Notes  WinObjC-only extension providing access
+         to UIModalPresentationFormSheet overlay.
+         Not exposed in header (clients must access
+         via performSelector).
+*/
+- (UIView*)_modalOverlayView {
+    return priv->_modalOverlayView;
+}
```

Comment:
I don't see this used anywhere; is it needed?  Also, we shouldn't be annotating internal/private methods.